clickhouse/base/wire_format.h

```diff
@@ -22,6 +23,7 @@ class WireFormat {
     static void WriteFixed(OutputStream& output, const T& value);
     static void WriteBytes(OutputStream& output, const void* buf, size_t len);
     static void WriteString(OutputStream& output, std::string_view value);
+    static void WriteQuotedString(OutputStream& output, std::optional<std::string_view> value);
```

Comment:
It looks like this function jams together both "write nullable string" semantics with "write quoted string", is there a way to do those in separate functions?